plugin/gudid-transformation-maven-plugin/src/main/java/dev/ikm/maven/FoiClassTransformer.java

```diff
@@ -124,7 +121,7 @@ public void transform(File inputFile, Composer composer) {
                             }
 
                             // Create session with ACTIVE state
-                            Session session = composer.open(State.ACTIVE, currentTime, author, module, path);
+                            Session session = composer.open(State.ACTIVE, author, module, path);
```

Comment:
Rather than creating one session per line in the txt file, if we move the session creation  above line 94, does that help performance? If so, I think we can safely do that.